### PR TITLE
Add a couple missing tkinter things

### PR DESCRIPTION
**Pull request by tungol**
The comment said this will cause lots of mypy problems, let's see what.

### CODE REVIEW

**Comment by tungol** [comment]
I see; the problem is that `config` also exists as an alias of `configure` in the base classes, and the signatures don't match. Mypy understands methods being overridden a lot better than it understands assignments being overridden. Hopefully this fixes it.

**Comment by tungol** [comment]
Too bad. Mypy and pyright have different behavior here. Given this:

```python
label = tkinter.Label()
reveal_type(label.config)
```

Mypy shows the more specific overload that created on `tkinter.Label`:
```
stdlib/@tests/test_cases/check_tkinter.py:34: note: Revealed type is "Overload(def (cnf: Union[builtins.dict[builtins.str, Any], None] =, *, activebackground: builtins.str =, activeforeground: builtins.str =, anchor: Union[Literal['nw'], Literal['n'], Literal['ne'], Literal['w'], Literal['center'], Literal['e'], Literal['sw'], Literal['s'], Literal['se']] =, background: builtins.str =, bd: Union[builtins.str, builtins.float] =, bg: builtins.str =, bitmap: builtins.str =, border: Union[builtins.str, builtins.float] =, borderwidth: Union[builtins.str, builtins.float] =, compound: Union[Literal['top'], Literal['left'], Literal['center'], Literal['right'], Literal['bottom'], Literal['none']] =, cursor: Union[builtins.str, tuple[builtins.str], tuple[builtins.str, builtins.str], tuple[builtins.str, builtins.str, builtins.str], tuple[builtins.str, builtins.str, builtins.str, builtins.str]] =, disabledforeground: builtins.str =, fg: builtins.str =, font: Union[builtins.str, tkinter.font.Font, builtins.list[Any], tuple[builtins.str], tuple[builtins.str, builtins.int], tuple[builtins.str, builtins.int, builtins.str], tuple[builtins.str, builtins.int, Union[builtins.list[builtins.str], builtins.tuple[builtins.str, ...]]], _tkinter.Tcl_Obj] =, foreground: builtins.str =, height: Union[builtins.str, builtins.float] =, highlightbackground: builtins.str =, highlightcolor: builtins.str =, highlightthickness: Union[builtins.str, builtins.float] =, image: Union[tkinter._Image, builtins.str] =, justify: Union[Literal['left'], Literal['center'], Literal['right']] =, padx: Union[builtins.str, builtins.float] =, pady: Union[builtins.str, builtins.float] =, relief: Union[Literal['raised'], Literal['sunken'], Literal['flat'], Literal['ridge'], Literal['solid'], Literal['groove']] =, state: Union[Literal['normal'], Literal['active'], Literal['disabled']] =, takefocus: Union[builtins.bool, Literal[0], Literal[1], Literal[''], def (builtins.str) -> Union[builtins.bool, None]] =, text: Union[builtins.float, builtins.str] =, textvariable: tkinter.Variable =, underline: builtins.int =, width: Union[builtins.str, builtins.float] =, wraplength: Union[builtins.str, builtins.float] =) -> Union[builtins.dict[builtins.str, tuple[builtins.str, builtins.str, builtins.str, Any, Any]], None], def (cnf: builtins.str) -> tuple[builtins.str, builtins.str, builtins.str, Any, Any])"
```

but pyright stays with the base class annotation and says:
```
stdlib/@tests/test_cases/check_tkinter.py:34:13 - information: Type of "label.config" is "(...) -> Any"
```

Given that, I don't know if there's a way to add `Misc.config` without causing problems.

File: stdlib/tkinter/font.pyi [hunk 1]
```diff
@@ -40,6 +41,7 @@ class _MetricsDict(TypedDict):
 class Font:
     name: str
     delete_font: bool
+    counter: itertools.count[int]
```
[the remark this answers is not in the document]
Comment (Akuli) [reply stdlib/tkinter/font.pyi:44]: Perhaps we should add a `# undocumented` comment here:

```suggestion
    counter: ClassVar[itertools.count[int]]  # undocumented
```

[the remark this answers is not in the document]
Comment (tungol) [reply stdlib/tkinter/font.pyi]: Thanks!

**Comment by Akuli** [comment]
> not sure if this actually matters to any type checkers? Seems more accurate though.

It doesn't, and we prefer leaving out the `= ...`. Apparently we don't have a linter rule for that.

**Review by Akuli — APPROVED** [review]
Thanks!

**Pull request merged**